client/src/context.rs

```diff
@@ -210,24 +203,17 @@ impl ContextOps for ClientContext {
 
         let params = CPUPOOL_INIT_PARAMS.with(|p| p.replace(None).unwrap());
 
-        let core = t!(core::spawn_thread("AudioIPC Client RPC", move || {
-            let handle = core::handle();
+        let core = core::spawn_thread("AudioIPC Client RPC", move || {
+            let handle = reactor::Handle::default();
```

Comment:
Is `reactor::Handle::default()` a corresponding handle of `current_thread::Handle`?

Reply:
They're different types of handles.

https://tokio.rs/docs/internals/net/#handle-current-vs-handle-default

Note that Handle::current() has since been deprecated because it's error-prone.